npm/src/lib/libDef.js

```diff
@@ -388,3 +398,81 @@ export async function getShallowGHLibDefs(
   }
   return _libdefs
 }
+
+const identity = i => i
+
+function getGHFlowVersionsForDef(def: ShallowGHLibDef): Promise<Array<Version>> {
```

Comment:
Need to call `_verifyCLIVersion()` in here too

(I'm going to send a commit to just add this in here myself -- just noting so I remember)